lib/stellate.rb

```diff
@@ -155,13 +155,16 @@ def self.use(schema, **kwargs)
   end
 
   def self.run_stellate_request(stellate_request)
-    url = URI.parse(stellate_request[:url])
+    url = URI.parse(stellate_request['url'])
+
     http = Net::HTTP.new(url.host, url.port)
+    http.use_ssl = true
+
     req = Net::HTTP::Post.new(url)
-    stellate_request[:headers].each do |key, value|
```

Comment:
Why did you decide to go away from symbols?

Reply:
Because we want to pass the `stellate_request` hash as argument to a sidekiq job, and sidekiq args must not contain symbols. (From my limited understanding, I guess it has something to do with serialization and how arguments are stored in Redis or whatever else storage you use with sidekiq.)